storybook/cypress/e2e/alert.cy.ts

```diff
@@ -20,7 +20,7 @@ describe("Alert", () => {
 
   for (const { status, message, icon } of statuses) {
     it(`should have appropriate message and icon for status "${status}"`, () => {
-      cy.get("dso-alert")
+      cy.get("dso-alert.hydrated")
```

Comment:
Waarom moet er nu .hydrated bij?

Reply:
Uitzoeken of custom `.get()` interessant is zodat altijd `.hydrated` wordt toegevoegd

Reply:
`cy.get("dso-alert .message")` wordt dat dan `cy.get("dso-alert.hydrated .message");` of laten we die met rust?

Afgesproken dat we `.get()` niet aanpassen en dat het een afspraak is dat we altijd `.hydrated` aan de web component selector toevoegen.